labs/lab-2-javascript-game/script.js

```diff
@@ -0,0 +1,47 @@
+function startGame() {
+  const play = prompt("Would you like to play?");
+  if (play === "yes") {
```

Comment:
Using `play.toLowerCase()` could let the user type "yes" or "Yes" (or "YES") and it would still work.